include/crm/common/errors_internal.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#ifndef PCMK__ERRORS_INTERNAL__H
+#define PCMK__ERRORS_INTERNAL__H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <glib.h>
+
+#include <crm/common/output_internal.h>
+
+/*!
+ * \internal
+ * \brief Handle end-of-program error reporting
+ *
+ * \param[in,out] error A GError object potentially containing some error.
+ * \param[in]     out   The output functions structure.
+ *
+ * \note \p error and \p out can both be NULL.
+ */
+void pcmk__cli_handle_error(GError *error, pcmk__output_t *out);
```

Comment:
I would just throw this in output_internal.h rather than add another header, since I don't see us adding much other error-related stuff, and this is output-related even if it doesn't require pcmk__output_t.

Also, I think the name could be misleading since the error is just output, not truly handled. It may only be used by CLI, but I think it's more generic itself. I'm thinking something like pcmk__output_and_free_error() to additionally indicate that error can't be used past that.